tests/gagarin/tuxlab.api.js

```diff
@@ -2,7 +2,7 @@
     using Meteor Gagarian (https://github.com/anticoders/gagarin)
 **/
 describe('tuxlab.task tests',function(){
-  var tuxlab = require('../../server/api/tuxlab.js');
```

Comment:
Please don't use relative paths.  Just declare the TuxLab package in the overall package.json, and then reference it by package name